molecules/_header.scss

```diff
@@ -0,0 +1,143 @@
+$header-height: $space-xl !default;
+$header-color: $white !default;
+$header-background-color: schema-color(5) !default;
+$header-accent-background-color: schema-color(6) !default;
+$header-border-color: schema-color(4) !default;
+
+@mixin level-1() {
+  background-color: $header-background-color;
+  color: $header-color;
+  line-height: $header-height;
+  position: relative;
+  z-index: 3;
+  box-shadow: $paper-shadow-2;
+}
+
+@mixin level-2() {
+  position: absolute;
+  z-index: 2;
+  background: $header-accent-background-color;
+  top: $header-height;
+  left: 0;
+  width: 100%;
+  height: $header-height * 2;
+  line-height: $header-height * 2;
+  transform: translate3d(0, -100%, 0);
+  transition: transform .15s linear;
+  overflow: hidden;
+  box-shadow: $paper-shadow-3;
+  &.is-active {
+    transform: translate3d(0, 0, 0);
+  }
+}
+
+@mixin main-menu() {
+  display: inline-block;
+  li {
+    @include list-item;
+  }
+}
+
+@mixin main-sub-menu() {
+  padding-left: $space-xxl;
+  height: 0;
+  visibility: hidden;
+  li {
+    display: inline-block;
+    opacity: 0;
+    transition: transform .1s linear;
+    transform: translate3d(150px, 0, 0);
+    color: $ghost;
+    @include list-item;
+    &:hover {
+      color: $white;
+    }
+  }
+  &.is-active {
+    height: auto;
+    visibility: visible;
+    li {
+      opacity: 1;
+      transform: translate3d(0, 0, 0);
+      transition: transform .25s cubic-bezier(0, 0.87, 0.37, 0.99), opacity .25s linear;
+    }
+    @for $i from 1 through 10 {
+      li:nth-child(#{$i}) {
+        transition-delay: $i * .05s + .075s;
+      }
+    }
+  }
+}
+
+@mixin list-item() {
+  font-weight: bold;
+  color: $miroschka;
+  &.is-current {
+    background-color: $header-accent-background-color;
+    color: $white;
+  }
+}
+
+.ws-header {
```

Comment:
I though we wrap all css parts around an @if $include-ws-header-css { because you might only want the mixins?